pupil-spa/src/app/survey-welcome/survey-welcome.component.html

```diff
@@ -0,0 +1,58 @@
+<app-header class="full-width"></app-header>
+
+<main id="content" class="js-content" role="main">
+  <div class="heading-block">
+      <header>
+          <div class="organisation-logo">
+              <div class="logo-text">Standards<br>&amp; Testing<br>Agency</div>
+          </div>
+      </header>
+  </div>
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <header>
+        <h1 class="heading-xlarge">Multiplication tables check ICT survey</h1>
+      </header>
+      <h3 class="heading-medium">
+        The purpose of this survey is to help us understand about 
+        information and communications technology (ICT) used in schools.
+      </h3>
+      <div class="panel panel-border-wide">
+        <p>
+          Complete this survey using your school ICT. If you have a range of devices, please run this survey on them one at a time.
+        </p>
+      </div>
+      <p>
+        Following the connection test, please give us feedback on ICT issues you may have come across. 
+      </p>
+      <p> 
+        You will be able to preview the multiplication tables check system after the connections test. 
+      </p>
+      <details>
+        <summary (click)="toggleDetails()"><span class="summary">{{ showDetails ? 'Less details' : 'More details' }}</span></summary>
```

Comment:
Surely this should be "Fewer details"?    Details can be counted accurately.  https://en.oxforddictionaries.com/usage/less-or-fewer

If we want to stick with "less", then perhaps "Less Detail" and "More detail" make sense.

Reply:
In the Axure, we have 'More detail' and 'Less detail'. Some places in gov.uk design system they don't change the text on the link https://design-system.service.gov.uk/components/details/ but in this case, it makes more sense to describe what the user will be doing by clicking on that link. I've raised it in the design systems github to see if anyone has opinions/research about this. For now let's keep it as 'More detail'/ 'Less detail'